src/ui/wxWidgets/PWSafeApp.cpp

```diff
@@ -271,7 +271,7 @@ bool PWSafeApp::OnInit()
   // Get the locale environment variable 'LC_CTYPE' specified by the environment
   // For instance, the behavior of function 'wcstombs' depends on the LC_CTYPE 
   // category of the selected C locale.
-  setlocale(LC_CTYPE, "");
+  setlocale(LC_CTYPE, "en_US.UTF-8");
```

Comment:
What is the advantage over the previous version?

Reply:
With my macOS Catalina the conversion from wxString (wchar_t) to UTF-8 coding is not running well as long as the setlocale function is not having the dedicated initialization. I do not know why, but this is the only chance that UTF-8 coding runs well.

Reply:
This could possibly lead to unexpected effects in other environments.
I would suggest using this as a fallback solution when "setlocale(LC_CTYPE, "")" fails to use the user’s preferred locale.

Can you run the "locale" command in a terminal to see what information it provides?